src/util.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Get the current totals.
+ *
+ * @param {Object} cart The cart object.
+ * @returns {Number} The totals of the cart object, if available, else the totals of the cart progress bar preview.
+ */
+export function getCurrentTotal( cart ) {
+	if ( undefined !== cart ) {
+		const total_price = cart.cartTotals.total_price;
+		const minor_unit = cart.cartTotals.currency_minor_unit;
+
+		return total_price / Math.pow( 10, minor_unit );
+	}
+	return 15;
+}
+
+/**
+ * Get the minor unit.
+ *
+ * @param {Object} cart The cart object.
+ * @returns {Number} The totals of the cart object, if available, else the totals of the cart progress bar preview.
+ */
+export function getMinorUnit( cart ) {
+	if ( undefined !== cart ) {
+		return cart.cartTotals.currency_minor_unit;
```

Comment:
nit, but to also protect against invalid cart response, you could write this function like so:

```
return cart?.cartTotals?.currency_minor_unit || '';
```

This will not throw an error if the cart is undefined, and will always default to an empty string should the minor unit be missing.

Reply:
Thanks for this great suggestion! I'll definitely use this pattern.